pygfx/utils/show.py

```diff
@@ -143,11 +143,11 @@ def show(
                 "Your scene does not contain any lights. Some objects may not be visible"
             )
 
-        existing_cameras = [x for x in scene.iter(lambda x: isinstance(x, Camera))]
+        existing_camera = next(scene.iter(lambda x: isinstance(x, Camera)), None)
```

Comment:
Wow,  I have never thought about using iterators this way ...

Reply:
:) It's their main selling point!